streampipes-client-python/docs/index.md

```diff
@@ -36,42 +36,58 @@ and the amazing universe of data analytics libraries in Python. </p>
 
 <br>
 
-**💡 The current version of this Python library is still a beta version.**
-<br>
-**This means that it is still heavily under development, which may result in frequent and extensive API changes, unstable behavior, etc.**
-<br>
+<p align="center"><b>💡 The current version of this Python library is still a beta version.<br>
+This means that it is still heavily under development, which may result in frequent and extensive API changes, unstable behavior, etc.</b>
+</p>
+---
+
+**🚧 Currently, we do not already version our Python documentation.
+Therefore, the provided docs always represent the development state.
+Please read our [getting started guide](./getting-started/first-steps.md) to find out how to install the development version of StreamPipes python.
+We will provide a versioned documentation as soon as possible. Stay tuned!ent, which may result in frequent and extensive API changes, unstable behavior, etc.**
```

Comment:
Something is wrong with the Stay tuned sentence ;-)

Reply:
Ah, damn
Thanks, there appears to be some leftover from the section above...

Reply:
Is adapted 🙂